config-reloader/datasource/kubedatasource/fluentdconfig.go

```diff
@@ -0,0 +1,223 @@
+package kubedatasource
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/vmware/kube-fluentd-operator/config-reloader/config"
+	kfoClient "github.com/vmware/kube-fluentd-operator/config-reloader/datasource/kubedatasource/fluentdconfig/client/clientset/versioned"
+	kfoInformers "github.com/vmware/kube-fluentd-operator/config-reloader/datasource/kubedatasource/fluentdconfig/client/informers/externalversions"
+	kfoListersV1beta1 "github.com/vmware/kube-fluentd-operator/config-reloader/datasource/kubedatasource/fluentdconfig/client/listers/logging.csp.vmware.com/v1beta1"
```

Comment:
Could we please change `logging.csp.vmware.com` to `logs.vdp.vmware.com`? Appreciate its a rather pedantic change but aligns more appropriately to our internal group names.

Reply:
Thanks for the feedback. I will try to get the change done either today or early next week.
I went with `logging.csp.vmware.com` as it is the prefix used by kube-fluentd-operator in the namespace annotations. Does it make sense to change those too to keep it consistent?

Reply:
It would make sense to change them and keep them consistent, but we'd just need to highlight it in the release notes of the next release to make sure people weren't relying on it to point to a different configmap name.